main.cpp

```diff
@@ -17,7 +20,7 @@ int main(int argc, char *argv[])
 
     QLocale::setDefault(QLocale(QLocale::English, QLocale::UnitedStates));
 
-    init_signal_handlers(argv[0]);
+    /* init_signal_handlers(argv[0]); */
```

Comment:
If needed un-comment it, otherwise remove it. Leaving it like this rises questions like: "Why is this here?"